chart/templates/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta2
+apiVersion: apps/v1
```

Comment:
Thanks @yonahd, is this retro compatible with older versions of Kubernetes though? 🤔

Reply:
Yes. It is available since v1.9. 
In 1.16 it is deprecated and can be enabled with a flag and will be completely removed in 1.18

Reply:
Ah very good to know, thanks! 🙇